src/svg_surface.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright 2015-2016, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use std::ffi::CString;
+use std::ops::Deref;
+use std::path::Path;
+
+extern crate libc;
+use libc::{c_char, c_double};
+
+use ffi;
+use ffi::enums::SurfaceType;
+
+use surface::{Surface, SurfaceExt};
+
+#[cfg(feature = "use_glib")]
+use glib::translate::*;
+
+pub struct SVGSurface(Surface);
+
+extern "C" {
+    pub fn cairo_svg_surface_create (filename: *const c_char,
```

Comment:
Shouldn't this be in the sys crate?

Reply:
I used the same code as PDFSurface

Reply:
Seems equally weird :) I meant in here btw https://github.com/gtk-rs/cairo/tree/master/cairo-sys-rs

Reply:
Yes, but we already allowed this in #143,
so IMHO we can ignore this, and fix with PDF yourself

Reply:
would it make any problems to move this code to the sys?

Reply:
I don't think so, why would it?

Reply:
No idea

Reply:
Ok tired to move them with out any problem, I will update it after lunch

Reply:
i updated it.